web/module/requests.py

```diff
@@ -22,20 +22,19 @@
     logger.debug('Current thread name: {}'. format(threading.current_thread().name))
 
     with data.Connection.use() as conn:
-        with data.Transaction(conn):
-            asyncio.sleep(0.1)
-            req = data.Request.get({'_id': req_id}, conn=conn).first()
-            return {
-                'responses': [
-                    {
-                        'type': 'return_value',
-                        'response_id': '0',
-                        'is_last': True,
-                        'result': {
-                            'machine_id': req.machine,
-                            'state': req.state,
-                            'type': req.type
-                        }
+        asyncio.sleep(0.1)
```

Comment:
This is just transaction removed, right?

Reply:
nope, must be here; a sync library is used in async environ.